### PR TITLE
added gcc option

### PR DESCRIPTION
**Pull request by Cvetomird91**
Added the option for "converting" C* source code into assembler instructions.

Provided we have source.c, the output will be written to a new file called source.s.

### CODE REVIEW

File: pages/common/gcc.md [hunk 1]
```diff
@@ -13,3 +13,7 @@
 - Include libraries from a different path:
 
 `gcc {{source.c}} -o {{executable}} -I{{header_path}} -L{{library_path}} -l{{library_name}}`
+
+- Export source code into Assembler instructions:
```
Comment (rubenvereecken) [inline pages/common/gcc.md]: I find "export" strange phrasing, or is it more common? I'd just use "compile to assembly"

**Comment by Cvetomird91** [comment]
I actually agree with you and I just changed "export" into "compile".

**Comment by rubenvereecken** [comment]
Quite happy with this :)

**Comment by Cvetomird91** [comment]
I'm glad to hear that.

**Pull request merged**